app/src/main/res/values-v14/dimens.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <dimen name="widget_margin">0dp</dimen>
+
+</resources>
```

Comment:
Our minimum sdk is above this?  I don't think this gets used at all and can be removed.

Reply:
Right, of course that makes sense.  I will remove it in my next commit